pytest_redis.py

```diff
@@ -54,19 +80,105 @@ def pytest_collection(session, genitems=True):
     return items
 
 
+def peek_next_item(iterable):
```

Comment:
Maybe use [LRANGE](http://redis.io/commands/LRANGE) instead so that you don't actually pop